### ISSUE TITLE
Accessibility: Mini Cart block button missing accessible name

### ISSUE THREAD

**Issue by webmandesign** [woocommerce/woocommerce-blocks#10284]
## Describe the bug

Checking my block theme powered WooCommerce website for accessibility issues with https://wave.webaim.org/ (and also Chrome's [Lighthouse accessibility audit](https://developer.chrome.com/docs/lighthouse/overview/)) I get an error of Mini Cart button containing no accessible name.

The issue only happens when the page is loaded as the block produces a button without any `aria-label` attribute.

When I hover over the Mini Cart button, its JavaScript kicks in and adds `aria-label` attribute dynamically.

However, the accessibility evaluation tools mentioned above work with JavaScript disabled, so you can reproduce the issue when you disable JavaScript in your browser too.

## To reproduce

1. Add a Mini Cart block (`woocommerce/mini-cart`) into site header, for example.
2. Disable JavaScript in your browser and inspect the Mini Cart button HTML using browser code inspector to see there is no text or `aria-label` set for the button.

## Expected behavior

Every button should have a label, text.

## Screenshots

![wc-mini-cart-a11y-issue](https://github.com/woocommerce/woocommerce-blocks/assets/2456735/d8e4639b-aa82-4cd3-9651-d289575d4cd1)

### WordPress (please complete the following information):

-   WordPress version: 6.2.2
-   WooCommerce version: 7.9.0

**Comment by nefeline** [comment]
Hi @webmandesign ! Thanks for reaching out to us and reporting this issue: I'm adding this to our backlog, more updates will be shared here as soon as we start working on it.

**Comment by webmandesign** [comment]
Meanwhile I've fixed this in my theme using code similar to:

```php
add_filter( 'render_block', function( $block_content, $block ) {

	if ( 'woocommerce/mini-cart' === $block['blockName'] ) {
		$html = new WP_HTML_Tag_Processor( $block_content );

		$html->next_tag( array( 'class_name' => 'wc-block-mini-cart__button' ) );
		$html->set_attribute( 'aria-label', __( 'Shopping cart', 'woocommerce' ) );

		$block_content = $html->get_updated_html();
	}

	return $block_content;

}, 10, 2 );
```
